### PR TITLE
relu kernel

### PR DESCRIPTION
**Pull request by AngelaZou2000**
Parallel relu kernel implemented.

### CODE REVIEW

File: hammerblade/torch/tests/test_relu.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 """
+
 import torch
 import torch.nn as nn
-import pytest
+from hypothesis import given
+from .hypothesis_test_util import HypothesisUtil as hu
 
-@pytest.mark.skip(reason="not yet implemented")
+# @pytest.mark.skip(reason="not yet implemented")
```
Comment (yodada) [inline hammerblade/torch/tests/test_relu.py]: these should be removed as well ;)

**Review by yodada — APPROVED** [review]
I think it looks good to me! simply remove those comments and it should be good to go

**Review by vb000 — APPROVED** [review]
LGTM! 

@AngelaZou2000 Interestingly, ReLU backward is also ReLU. Consider adding backward tests as well -- you implemented 2 kernels ;)

**Comment by cbatten** [comment]
@AngelaZou2000 great work! Your first approved pull request -- a great milestone!

File: hammerblade/torch/tests/test_relu.py (hunk 1 above)
Comment (AngelaZou2000) [reply hammerblade/torch/tests/test_relu.py]: okie I'll do that right now!

**Comment by yodada** [comment]
@AngelaZou2000 have you tried cosim?

**Comment by AngelaZou2000** [comment]
> @AngelaZou2000 have you tried cosim?

yea but I think I might did something wrong sourcing the setup scripts?

**Pull request merged**